etomica-core/src/main/java/etomica/api/IBox.java

```diff
@@ -4,16 +4,27 @@
 
 package etomica.api;
 
+import java.util.List;
+
+/**
+ * IBox holds a boundary and a list of molecules.  The boundary defines the
+ * spatial extent of the box and conditions at the edges of the boundary.  The
+ * molecules in the box interact via potentials, but do not interact with
+ * molecules in other boxes.  IBox contains methods to add and remove molecules
+ * and an event manager that notifies listeners about those events.
+ */
 public interface IBox {
 
     /**
      * Informs the IBox what its index is.  This should only be called by the
      * ISimulation.
+     *
+     * @param newIndex the box's new index
      */
-    public void setIndex(int newIndex);
+    public List setIndex(int newIndex);
```

Comment:
Ah, I see. I suspected you fixed it, but I haven't figured out yet how to see the big picture in github.